src/Components/MainHeader/MiddleMenu/Function/NOP/Nop.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import './Nop.css';
+
+function Add() {
+  const [number, setNumber] = useState(0);
+  const onIncrease = () => {
+    setNumber((prevNumber) => prevNumber + 1);
```

Comment:
```suggestion
    setNumber(prevNumber => prevNumber + 1);
```

```suggestion
    setNumber(number + 1);
```
의 차이를 명확히 이해할 수 있어야 합니다